test/package.json

```diff
@@ -12,6 +12,7 @@
     "watch": "npm run watch:build & sleep 5 && npm run watch:process-inspect server",
     "build": "soundworks-template-build --build",
     "clean": "soundworks-template-build --clean",
+    "check-types": "soundworks-template-build --check-types",
```

Comment:
Note: Basically, there is `tsconfig.json` file needed for this call. Since it is project dependent, I've dropped this for for the template-build project.